live-build/misc/migration-scripts/dx_apply

```diff
@@ -139,6 +139,10 @@ zfs create \
 	"$RPOOL/ROOT/$FSNAME/home" ||
 	die "failed to create linux dataset $RPOOL/ROOT/$FSNAME/home"
 
+#
+# Note that this dataset should not contain any useful data as it will be
+# replaced by a clone of the current /var/delphix dataset in dx_execute.
```

Comment:
Should we also explain "why?" here?
(e.g. have not quiesced VDB yet, there can be configuration changes, etc..)